web/themes/contrib/civictheme/tests/src/Kernel/CivicthemeMediaUtilityKernelTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Drupal\Tests\civictheme\Kernel;
```

Comment:
@joshua-salsadigital Could you review the kernel test - as we fix bugs in utility functions I would like to add kernel tests as these are more appropriate for testing these utilities than behat.